channeldb/channel.go

```diff
@@ -37,6 +37,11 @@ var (
 	// TODO(roasbeef): flesh out comment
 	openChannelBucket = []byte("open-chan-bucket")
 
+	// closedChannelBucket stores all channels that have seen their
+	// commitment tx confirm. All information from their previous open state
+	// is retained.
+	closedChannelBucket = []byte("confirmed-chan-bucket")
```

Comment:
nit: "confirmed" is ambiguous. to me it sounds like the transition from pending->open. perhaps historical-chan-bucket?

Reply:
renamed